ironfish/src/rpc/adapters/tlsAdapter.ts

```diff
@@ -4,29 +4,34 @@
 import net from 'net'
 import { pki } from 'node-forge'
 import tls from 'tls'
+import { v4 as uuid } from 'uuid'
 import { FileSystem } from '../../fileSystems'
 import { createRootLogger, Logger } from '../../logger'
+import { IronfishNode } from '../../node'
 import { ApiNamespace } from '../routes'
 import { RpcSocketAdapter } from './socketAdapter/socketAdapter'
 
 export class RpcTlsAdapter extends RpcSocketAdapter {
   readonly fileSystem: FileSystem
   readonly nodeKeyPath: string
   readonly nodeCertPath: string
+  node: IronfishNode
 
   constructor(
     host: string,
     port: number,
     fileSystem: FileSystem,
     nodeKeyPath: string,
     nodeCertPath: string,
+    node: IronfishNode,
     logger: Logger = createRootLogger(),
     namespaces: ApiNamespace[],
   ) {
-    super(host, port, logger, namespaces)
+    super(host, port, logger, namespaces, true)
```

Comment:
I don't really know what this true is, it would be a bit cleaner if it just over-rode authentication below on line 35.